crates/turbopack-bench/src/util/mod.rs

```diff
@@ -70,7 +70,7 @@ pub fn build_test(module_count: usize, bundler: &dyn Bundler) -> TestApp {
     .unwrap();
 
     let npm = command("npm")
-        .args(["install", "--prefer-offline", "--loglevel=error"])
+        .args(["install", "--loglevel=error"])
```

Comment:
prefer-offline does not work reliably, it ends up causing resolving of dependencies to fail with an error saying that a package version does not exist that does exist on the npm registry.